nx-X11/programs/Xserver/Imakefile

```diff
@@ -296,11 +296,19 @@ ZLIB      =  $$(pkg-config --exists 'zlib' >/dev/null 2>&1 && pkg-config --libs
 PNGLIB    =  `pkg-config --libs libpng`
 JPEGLIB   =  -ljpeg
 
+TIRPC_LIB =
+
+#if HasSecureRPC
+#if UseTIRPC
+TIRPC_LIB =  $$(pkg-config --libs 'libtirpc')
+#endif
+#endif
+
 NXAGENT = hw/nxagent/LibraryTargetName(nxagent)
 NXCOMPEXT =  hw/nxagent/compext/LibraryTargetName(compext)
 NXAGENTLIBS = PreFbLibs $(NXAGENT) FbPostFbLibs $(NXCOMPEXT) $(NXAGENT) $(MI)
 NXAGENTSYSLIBS = $(LDPRELIBS) $(XLIB) $(SYSLIBS) $(XPMLIB) $(XMLLIB)
-NXCOMPEXTSYSLIBS = $(PIXMANLIB) $(ZLIB) $(PNGLIB) $(JPEGLIB)
+NXCOMPEXTSYSLIBS = $(PIXMANLIB) $(ZLIB) $(PNGLIB) $(JPEGLIB) $(TIRPC_LIB)
```

Comment:
The TIRPC_LIB is not required for nxcompext build. Thus, it should be moved up and added to NXAGENTSYSLIBS. IMHO.

Reply:
Neither is `pixman` as far as I can tell (though definitely a dependency of `nxagent`.) I'll try moving both and see what happens.